### ISSUE TITLE
License missing

### ISSUE THREAD

**Issue by andreacivita** [francescomalatesta/wolframalphaphp#5]
Project has no license.

**Comment by andreacivita** [comment]
After several months, there's no update about this issue.

As said in Github docs:

`You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. If you're creating an open source project, we strongly encourage you to include an open source license. The Open Source Guide provides additional guidance on choosing the correct license for your project.`

In short, without license people has no right to  edit and redistribute source code in this repo.
The only right we have is to fork or view code, but nothing else.

(https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license)

So, I'm going to close this issue.
